Jenkinsfile

```diff
@@ -1,19 +1,19 @@
 pipeline {
     agent {
-        node { label 'Jekyl' }
+        node { label 'Jekyll' }
```

Comment:
I don't know this syntax, so bear with me... Doesn't this mean we need to update labels in all Jenkins instances, to match the (now correctly) spelled name?

https://jenkins.io/doc/book/pipeline/syntax/ <-- Syntax reference in case someone else is new at this

Reply:
I did update the labels in the PELUX instance, so it shouldn't be a biggie.

Reply:
But yes, it does mean that. The software used is called Jekyll however, so better fix it now before it becomes too difficult to change.